packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -81,12 +82,13 @@ export default class FreeDrawLayer extends PureComponent {
 
   @autobind
   enableMapDraw() {
```

Comment:
can you rename this `enableMapDraw` method to `handleShapeDrawing`? I just noticed that it seemed like the same as the prop and it's slightly confusing.

Reply:
To Anye's point, it is potentially confusing.